shell/key-bindings.zsh

```diff
@@ -10,4 +10,10 @@
   }
   zle     -N     tome-show-widget
   bindkey '^R^R' tome-show-widget
+
+  tome-write-command() {
+    selected=$(fc -lrn 0 | fzf | sed 's/\\n/\n/g')
```

Comment:
I'd prefer using `zle` to fetch the selected command from the history. This way you get the raw command and don't have to replace any escaped newlines. 

see https://github.com/junegunn/fzf/blob/master/shell/key-bindings.zsh#L106